source/getting-started/js-primer.md

```diff
@@ -52,3 +52,50 @@ export default Ember.Component.extend({
   }
 });
 ```
+
+## Object Literal Shorthand
+
+In ES6 some functionality was added that allows the use of terser syntax for object literal properties and methods, along with some enhanced functionality for property names.
+
+### Property intializer syntax
+
+Now properties can be set on object literals without by including the name of the property without the typical colon and value if the object property the same as the local variable you intend to set as the value of that property. This helps reduce duplication. 
```

Comment:
Not sure we need `Now` to begin the sentence.

Reply:
Also, see if you can provide example of how it's done currently:
```
function someFunc(thing1, thing2) {
  return {
    thing1: thing1,
    thing2: thing2
  };
}
```
then move on to describe how the new syntax makes it simpler. Just so the learner doesn't have to conjure things up in their mind. Also remind them that it doesn't apply in cases where object key isn't equal to value. And that both old and new style can be mixed.